aptos-move/aptos-gas/src/aptos_framework.rs

```diff
@@ -7,16 +7,70 @@ crate::natives::define_gas_parameters_for_natives!(GasParameters, "aptos_framewo
     [.account.create_address.base_cost, "account.create_address.base", 1],
     [.account.create_signer.base_cost, "account.create_signer.base", 1],
 
+    [.signature.bls12381_aggregate_pop_verified_pubkeys.base_cost, "signature.bls12381_aggregate_pop_verified_pubkeys.base", 1],
```

Comment:
How much do these newly added functions cost in terms of computing resources compared to ed25519? Asking so that we can decide the appropriate gas costs for these later.

Reply:
I have some benchmarks on Notion under the Research page, I'll send it to you.

Reply:
cc @vgao1996